src/widgets/DateRangeSelector.js

```diff
@@ -38,9 +38,7 @@ export const DateRangeSelector = ({
         <DatePickerButton
           minimumDate={minimumDate}
           maximumDate={initialEndDate}
-          initialValue={moment(initialStartDate)
-            .startOf('day')
-            .toDate()}
+          initialValue={moment(initialStartDate).startOf('day').toDate()}
```

Comment:
whats going on 😨

Reply:
Linter was unhappy so I just committed it 🤷‍♂️